crates/rattler_shell/src/shell/mod.rs

```diff
@@ -422,12 +422,22 @@ impl Shell for Fish {
     }
 }
 
+fn escape_backslashes(s: &str) -> String {
+    s.replace('\\', "\\\\")
+}
+
 /// A [`Shell`] implementation for the Bash shell.
 #[derive(Debug, Clone, Copy, Default)]
 pub struct NuShell;
 
 impl Shell for NuShell {
     fn set_env_var(&self, f: &mut impl Write, env_var: &str, value: &str) -> std::fmt::Result {
+        // escape backslashes for Windows (make them double backslashes)
+        let value = if cfg!(windows) {
+            escape_backslashes(value)
+        } else {
+            value.to_string()
+        };
```

Comment:
Dont you always want to escape these? Its not windows specific but more how nushell interprets a string right?